depscan/cli.py

```diff
@@ -744,6 +744,16 @@ def main():
         pkg_vulnerabilities,
         pkg_group_rows,
     ) = (None, None, None, None, None, None)
+    if os.getenv("GITHUB_ACTIONS").lower() == "__appthreat_dep-scan-action" \
+        and not os.getenv("INPUT_THANK_YOU") == ("I have sponsored "
+                                                 "OWASP-dep-scan."):
+        LOG.error("You have not indicated that you have sponsored "
```

Comment:
May be info within a rich panel?

Reply:
@prabhu Is this ok? Wasn't sure about title.

![image](https://github.com/owasp-dep-scan/dep-scan/assets/80227828/38ad5038-22eb-4d01-bcc1-becfa79b3293)